keras_nlp/models/bert/bert_preprocessing.py

```diff
@@ -167,15 +168,27 @@ class BertPreprocessor(keras.layers.Layer):
 
     This preprocessing layer will do three things:
 
-     - Tokenize any number of inputs using the `tokenizer`.
+     - Tokenize any number of input segments using the `tokenizer`.
```

Comment:
Should we add a note about the ambiguous 
```python
x=(sentence1, sentence2)
``` 
input? Maybe we want to be explicit about how to get two segments in one example?

Reply:
Added a few usage examples about this. Will land as is just to unblock folk, but we can keep updating docs in a follow up!